packages/mui-material-next/src/Slider/Slider.types.ts

```diff
@@ -0,0 +1,310 @@
+import * as React from 'react';
+import { SlotComponentProps } from '@mui/base';
+import { Mark } from '@mui/base/useSlider';
+import { SxProps } from '@mui/system';
+import { OverridableStringUnion, OverrideProps, OverridableComponent } from '@mui/types';
+import { Theme } from '../styles';
+import SliderValueLabelComponent from './SliderValueLabel';
+import { SliderClasses } from './sliderClasses';
+
+export interface SliderPropsColorOverrides {}
+
+export interface SliderPropsSizeOverrides {}
+
+export interface SliderComponentsPropsOverrides {}
+
+export interface SliderOwnerState extends SliderProps {
+  dragging: boolean;
+  marked: boolean;
+  focusedThumbIndex: number;
+}
+
+export interface SliderTypeMap<D extends React.ElementType = 'span', P = {}> {
+  props: P & {
+    /**
+     * The label of the slider.
+     */
+    'aria-label'?: string;
+    /**
+     * The id of the element containing a label for the slider.
+     */
+    'aria-labelledby'?: string;
+    /**
+     * A string value that provides a user-friendly name for the current value of the slider.
+     */
+    'aria-valuetext'?: string;
+    /**
+     * The color of the component.
+     * It supports both default and custom theme colors, which can be added as shown in the
+     * [palette customization guide](https://mui.com/material-ui/customization/palette/#adding-new-colors).
+     * @default 'primary'
+     */
+    color?: OverridableStringUnion<'primary' | 'secondary', SliderPropsColorOverrides>;
+    /**
+     * The components used for each slot inside.
+     *
+     * This prop is an alias for the `slots` prop.
+     * It's recommended to use the `slots` prop instead.
+     *
+     * @default {}
+     */
+    components?: {
```

Comment:
I'm not sure if they were any discussions around this, but I think we can drop the `components` and `componentsProps` props in the new components as they are deprecated. cc @mnajdova

Reply:
You're right! it was decided that we would drop them, I just forgot to do it 😅 Updated the PR to remove these and modify the tests accordingly. I think we can also remove these tests as they would be covered by [`describeConformance`](https://github.com/mui/material-ui/blob/4bfc582c39248648200b03bb664086686cff1a5b/packages/mui-material-next/src/Slider/Slider.test.js#L45-L86), right?:
- [slots test](https://github.com/mui/material-ui/blob/4bfc582c39248648200b03bb664086686cff1a5b/packages/mui-material-next/src/Slider/Slider.test.js#L1342-L1357)
- [slotProps test](https://github.com/mui/material-ui/blob/4bfc582c39248648200b03bb664086686cff1a5b/packages/mui-material-next/src/Slider/Slider.test.js#L1359-L1373)